coreJava/src/main/java/com/brandon/corejava/utilities/CommonActivityFunctions.java

```diff
@@ -0,0 +1,38 @@
+package com.brandon.corejava.utilities;
+
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.util.Log;
+import android.view.View;
+import android.widget.TextView;
+
+import com.google.android.material.textfield.TextInputEditText;
+
+import java.util.function.Consumer;
+
+public interface CommonActivityFunctions {
+
+    default void onTextChanged(TextInputEditText editText, Consumer<String> lambda) {
```

Comment:
No need to make an interface with default methods like this. What is this solving? If you want common functions make them static

Name the second variable something descriptive like `onTextChanged`

Reply:
Went ahead and made this a static class, my intent originally was to try to keep it limited to being used where the interface was used, but I can see how that would lead to some weirdness if I wanted to use it without using the interface.